src/Workspaces/Core/Portable/Shared/Utilities/SymbolEquivalenceComparer.GetHashCodeVisitor.cs

```diff
@@ -167,7 +167,22 @@ private int CombineHashCodes(IModuleSymbol x, int currentHash)
 
             private int CombineHashCodes(INamedTypeSymbol x, int currentHash)
             {
-                return Unwrap(x).Aggregate(currentHash, (a, n) => CombineNamedTypeHashCode(n, a));
+                currentHash = CombineNamedTypeHashCode(x, currentHash);
```

Comment:
Add a note about perf optimization so that people don't go about simplifying the code again?